README.md

```diff
@@ -330,6 +331,24 @@ end
 
 An optional `tracer` named argument can be provided to use a custom tracer. It will default to `OpenTracing.global_tracer` if not provided.
 
+## PostgreSQL
+
+PostgreSQL (pg) instrumentation traces all queries performed with the ruby-pg client.
```

Comment:
ruby-pg has been eol for some time.

Reply:
The gem `https://rubygems.org/gems/pg` that links to `ged/ruby-pg`

Reply:
That's the github project name, but we are referring to libraries by their gem.